partitura/io/exportmei.py

```diff
@@ -295,6 +296,22 @@ def _handle_tuplets(self, measure_el, start, end):
         for tuplet in self.part.iter_all(spt.Tuplet, start=start, end=end):
             start_note = tuplet.start_note
             end_note = tuplet.end_note
+            if start_note.start.t < start or end_note.end.t > end:
+                warnings.warn(
+                    "Tuplet start or end note is outside of the measure. Skipping tuplet element."
+                )
+                continue
+            if start_note.start.t > end_note.start.t:
+                warnings.warn(
+                    "Tuplet start note is after end note. Skipping tuplet element."
+                )
+                continue
+            # Skip if start and end notes are in different voices or staves
+            if start_note.voice != end_note.voice or start_note.staff != end_note.staff:
```

Comment:
why should we skip if we are changing staff? This could be a staff crossing voice once we fix the modeling problem of partitura with these things.
I would allow it, unless it creates problems with the actual version

Reply:
This is true. However, it created some problems with the export. This is quite tricky. The issue comes from the Tuplet objects which should have tests implemented on them. Well, I can either comment out the staff and add a NOTE that this might create errors or add it as a flag on the function (i.e. cross_staff_in_tuplets=True).